tackle-test-generator-unit/pom.xml

```diff
@@ -35,8 +29,8 @@
     <maven-jar-plugin.version>3.2.0</maven-jar-plugin.version>
     <maven-javadoc-plugin.version>3.0.0</maven-javadoc-plugin.version>
     <coveralls-maven-plugin.version>4.3.0</coveralls-maven-plugin.version>
-	<evosuite.version>1.2.0</evosuite.version>
-    <randoop.version>4.3.0</randoop.version>
+	  <evosuite.version>v1.2.0</evosuite.version>
```

Comment:
why added v to the version numbers for randoop and evosuite?

Reply:
@rachelt44: That's how evosuite and randoop are available on jitpack:
- https://jitpack.io/#evosuite/evosuite/v1.2.0
- https://jitpack.io/#randoop/randoop/v4.3.0